src/Stripe.net/Services/_base/StringEnum.cs

```diff
@@ -0,0 +1,17 @@
+namespace Stripe
+{
+    public abstract class StringEnum
```

Comment:
You may not have done it because this is still a prospective change, but before merging could you add some documentation on this class that talks a little about why it exists?